dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -579,7 +604,7 @@
                 <ControlTemplate TargetType="primitives:NavigationViewItemPresenter">
                     <Grid
                         x:Name="LayoutRoot"
-                        Height="40"
+                        MinHeight="{ThemeResource NavigationViewItemOnLeftMinHeight}"
```

Comment:
You changed from Height to MinHeight. Expected?

Reply:
yes.